config/locales/en.yml

```diff
@@ -1586,8 +1594,12 @@ en:
       account: Public posts from @%{acct}
       tag: 'Public posts tagged #%{hashtag}'
   scheduled_statuses:
-    over_daily_limit: You have exceeded the limit of %{limit} scheduled posts for today
-    over_total_limit: You have exceeded the limit of %{limit} scheduled posts
+    over_daily_limit:
+      one: You have exceeded the limit of %{limit} scheduled post for today
+      other: You have exceeded the limit of %{limit} scheduled posts for today
+    over_total_limit:
+      one: You have exceeded the limit of %{limit} scheduled post
+      other: You have exceeded the limit of %{limit} scheduled posts
```

Comment:
Same thing with `limit` VS `count`, and same thing with the limit being a hardcoded value (currently 25).